driver-core/src/main/com/mongodb/client/model/Accumulators.java

```diff
@@ -36,6 +36,23 @@
  */
 public final class Accumulators {
 
+    /**
+     * Gets a field name for a $group operation representing the number of documents in the group.
+     *
+     * <p>
+     * {@code Accumulators.count("count")} has equivalent semantics to {@code Accumulators.sum("count", 1)}.
+     * </p>
+     *
+     * @param fieldName the field name
+     * @return the field
+     * @since 4.3
+     * @mongodb.server.release 5.0
+     * @mongodb.driver.manual reference/operator/aggregation/count/ $count
```

Comment:
This link is the same as `Aggregates.count` which doesn't reference $group at all? And says server version 3.4 (which Aggregates.count also says). Do you know if this will be updated for 5.0 ?
(https://docs.mongodb.com/v5.0/reference/operator/aggregation/count/)

Its a shame and becomes confusing when the same operator can be used in multiple stages which require different syntax.

For example: $sum can now be used in $group, $project, $addFields, $set, $replaceRoot, $replaceWith and $match when used with $expr - not sure if it always has the same syntax.

Reply:
I suggest adding the json form of what this count produces, top help highlight why its different to `Aggregates.count`:

Reply:
I don't know if it will be updated.  I can open a ticket though.  We could link to https://docs.mongodb.com/v5.0/reference/operator/aggregation/count-accum/#mongodb-group-grp.-count, but from what I can see $count is the only accumulator that has a URI of this form.

Reply:
Good spot - yeah linking to https://docs.mongodb.com/v5.0/reference/operator/aggregation/count-accum/ makes sense

Reply:
Updated docs links.

Reply:
> I suggest adding the json form of what this count produces, top help highlight why its different to Aggregates.count

I considered that but then it would be the only builder method that had such an example.  I think the docs link that shows it should be sufficient.

Reply:
Ack